tests/phpunit/tests/html-api/wpHtmlProcessorSemanticRules.php

```diff
@@ -392,4 +392,29 @@ public function test_in_body_any_other_end_tag_with_unclosed_non_special_element
 		$this->assertSame( 'DIV', $p->get_tag(), "Expected to find DIV element, but found {$p->get_tag()} instead." );
 		$this->assertSame( array( 'HTML', 'BODY', 'DIV', 'DIV' ), $p->get_breadcrumbs(), 'Failed to produce expected DOM nesting: SPAN should be closed and DIV should be its sibling.' );
 	}
+
+	/**
+	 * Ensures that support isn't accidentally partially added for the closing BR tag `</br>`.
+	 *
+	 * This tag closer has special rules and support shouldn't be added without implementing full support.
+	 *
+	 * > An end tag whose tag name is "br"
+	 * >   Parse error. Drop the attributes from the token, and act as described in the next entry;
+	 * >   i.e. act as if this was a "br" start tag token with no attributes, rather than the end
+	 * >   tag token that it actually is.
+	 *
+	 * When this handling is implemented, this test should be removed and `test_br_end_tag_special_behavior`
+	 * should not be marked incomplete. It's not incorporated into the existing unsupported tag behavior test
+	 * because the opening tag is supported; only the closing tag isn't.
```

Comment:
This mentions an incomplete test (by name `test_br_end_tag_special_behavior`) that we've removed. Let's remove the specific reference to that test.

Reply:
thanks! good spotting. I've removed it.